integrationtest/src/main/java/software/amazon/timestream/integrationtest/Constants.java

```diff
@@ -56,6 +50,51 @@ final class Constants {
       "measure_value::varchar",
   };
 
+  static final String NO_DB_NO_TB_REGION = "ap-southeast-2";
+  static final String[] NO_DB_DATABASE_NAMES = new String[]{};
+  static final String[] NO_TB_TABLE_NAMES = new String[]{};
```

Comment:
Add a new line to separate different group of data.

Reply:
sure, fixed.